app/templates/docs/scim.html

```diff
@@ -0,0 +1,422 @@
+<ul>
+  <li>Required:
+    <ul>
+      <li>one of displayName, name.formatted, or name.givenName and name.familyName</li>
+      <li>emails.value</li>
+    </ul>
+  </li>
+  <li>Optional:
+    <ul>
+      <li>userName</li>
+      <li>Normally best to omit, and one will be autogenerated. If supplied, must be 3-100 characters of only lowercase letters, underscores and numbers.</li>
+      <li>locale</li>
+      <li>timezone</li>
+    </ul>
+  </li>
+</ul>
+
+<p>If successful, the response will have a status of 201, and contain the full User resource in the body.</p>
+<p>The created Trello user will be owned by the enterprise. If the enterprise has domains configured, the user will not email address require confirmation. Otherwise, an email will be sent asking the user to confirm their address.</p>
+<p>At present, it is not possible to assign a user to any Teams or Boards at creation. This may be supported in a later release of the Trello SCIM API.</p>
+
+<h3 id="update-a-user">Update a User</h3>
+<p>Currently, you can only update a user's active attribute.</p>
+<p>When a user is set to active: false, they will be deactivated from all enterprise teams and boards. If the user is then set to active: true, they will be reactivated on any teams and boards they had previously been deactivated from when deactivated from the enterprise (via the SCIM API or Enterprise Admin Dashboard).</p>
+<p>If the user had been deactivated from each teams individually, setting them to active: true via the SCIM API will reactivate them in all the enterprise teams they were previously a member of.</p>
+<p>A user can be updated via PATCH or PUT requests. Remember to send a Content-type header with application/json or application/scim+json.</p>
+
+<h4>Updating via PATCH</h4>
+<div class="highlight-URL">
+  <div class="highlight">
+    <pre>PATCH https://api.trello.com/scim/v2/Users/58d11883e7ea028acc5b19a4&amp;key=[application_key]&amp;token=[token]</pre>
+  </div>
+</div>
+<div class="highlight"><pre><code class="json">{
+  "schemas": ['urn:ietf:params:scim:api:messages:2.0:PatchOp']
+  "Operations": [ { "op": "replace", "value": { "active": false } } ]
+}
+</code></pre></div>
+
+<h4>Updating via PUT</h4>
+<div class="highlight-URL">
+  <div class="highlight">
+    <pre>PUT https://api.trello.com/scim/v2/Users/58d11883e7ea028acc5b19a4&amp;key=[application_key]&amp;token=[token]</pre>
+  </div>
+</div>
+<div class="highlight"><pre><code class="json">{
+  "schemas": [
+    "urn:ietf:params:scim:schemas:core:2.0:User"
+  ],
+  "displayName": "Little Bobby Tables",
+  "name": { "formatted": "Little Bobby Tables" },
+  "userName": "bobby_tables",
+  "active": false,
+  "emails": [
+    {
+      "value": "bobby_tables@example.com",
+      "primary": true
+    }
+  ]
+}
+</code></pre></div>
+
+
+<p>Whilst at present everything other than the active attribute is ignored, in future we may support updating other attributes via PUT.</p>
```

Comment:
“active” here should definitely be in an inline code block, otherwise it reads like an adjective of “attribute”